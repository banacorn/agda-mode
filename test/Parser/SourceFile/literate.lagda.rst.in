This line is ordinary text, which is ignored by Agda.

::

  module Whatever where
  -- Agda code goes here

Another non-code line.
::
.. This line is also ignored