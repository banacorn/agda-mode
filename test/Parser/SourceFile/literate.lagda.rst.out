Hole [0] (112, 113) => (112, 119) "{!   !}"